docs/ballerina-by-example/examples/byte-i-o/byte-i-o.bal

```diff
@@ -1,72 +1,79 @@
 import ballerina.io;
 
-@Description{value:"This function returns a ByteChannel from a given file location according to the specified file permission (whether the file should be opened for reading/writing)."}
+@Description {value:"This function returns a ByteChannel from a given file location according to the specified file permission (whether the file should be opened for reading/writing)."}
 function getFileChannel (string filePath, string permission) (io:ByteChannel) {
     // Here is how the ByteChannel is retrieved from the file.
     io:ByteChannel channel = io:openFile(filePath, permission);
     return channel;
 }
 
-@Description{value:"This function reads the specified number of bytes from the given channel."}
-function readBytes (io:ByteChannel channel, int numberOfBytes) (blob, int) {
+@Description {value:"This function reads the specified number of bytes from the given channel."}
+function readBytes (io:ByteChannel channel, int numberOfBytes, int offset) (blob, int) {
     blob bytes;
     int numberOfBytesRead;
+    io:IOError readError;
     // Here is how the bytes are read from the channel.
-    bytes, numberOfBytesRead = channel.readBytes(numberOfBytes);
+    bytes, numberOfBytesRead, readError = channel.read(numberOfBytes, offset);
```

Comment:
Shall we user var here?

Reply:
@anupama-pathirage i do agree that using var will help reduce the number of lines, apart from reducing the number of lines is there any other reason to use var instead of type ? 

(the rationale for using the types is that while looking at the example it will be clear for a developer on the return types of a function, var sounds like a generic form which might make the code less self explanatory) correct me if i am wrong here :)

Reply:
As discussed with @anupama-pathirage we will be leaving the samples as it is to preserve readability instead of using var